src/cmdlib.sh

```diff
@@ -49,6 +64,8 @@ prepare_build() {
     preflight
     if ! [ -d repo ]; then
         fatal "No $(pwd)/repo found; did you run coreos-assembler init?"
+    elif ! has_privileges && [ ! -f cache.qcow2 ]; then
```

Comment:
maybe we should make the name of this file a variable so that we can easily change it if we need to